README.md

```diff
@@ -14,8 +14,9 @@ This project hosts the .NET client library for the Google Ads API.
 
 ## Requirements
```

Comment:
It would be better to change the heading to "Supported Frameworks" and leave the bullet points as-is.
.NET Core 2.0 support is a given from .NET Standard 2.0 support